IronPdf.SmokeTests/src/test/java/com/ironsoftware/ironpdf/PdfATests.java

```diff
@@ -1,5 +1,7 @@
 package com.ironsoftware.ironpdf;
 
+import com.ironsoftware.ironpdf.render.ChromePdfRenderOptions;
+import com.ironsoftware.ironpdf.render.TableOfContentsTypes;
```

Comment:
this line is not necessary right?

Reply:
I don't think so. I will removed that out